docs/integration.rst

```diff
@@ -65,3 +65,44 @@ You can configure yamllint to run on ``arc lint``. Here is an example
       }
     }
   }
+
+Intergration with GitLab
+------------------------
+
+You can use the following gitlab-ci stage to do run yamllint and get the results as a
+`Code quality (Code Climate) <https://docs.gitlab.com/ee/ci/testing/code_quality.html>` report.
+
+.. code:: yaml
+
+   ---
+   lint:
+     stage: lint
+     script:
+       - pip install yamllint
+       - mkdir reports
+       - >
+         yamllint -f parsable . | tee >(awk '
+         BEGIN {FS = ":"; ORS="\n"; first=1}
+         {
+             gsub(/^[ \t]+|[ \t]+$|"/, "", $4);
+             match($4, /^\[(warning|error)\](.*)\((.*)\)$/, a);
+             sev = (a[1] == "error" ? "major" : "minor");
+             if (first) {
+                 first=0;
+                 printf("[");
+             } else {
+                 printf(",");
+             }
+             printf("{\"location\":{\"path\":\"%s\",\"lines\":{\"begin\":%s,\"end\":%s}}," \
+                    "\"severity\":\"%s\",\"check_name\":\"%s\",\"categories\":[\"Style\"]," \
+                    "\"type\":\"issue\",\"description\":\"%s\"}",
+                    $1, $2, $3, sev, a[3], a[2]);
```

Comment:
Those 4 lines would make yamllint report `line-length` errors, which is a bit ironic.

I propose this instead:
```suggestion
             printf("{\"location\":{\"path\":\"%s\",\"lines\":{\"begin\":%s",\
                    "\"end\":%s}},\"severity\":\"%s\",\"check_name\":\"%s\","\
                    "\"categories\":[\"Style\"],\"type\":\"issue\","\
                    "\"description\":\"%s\"}", $1, $2, $3, sev, a[3], a[2]);
```

Reply:
Ah good catch, I had overwritten the line-length settings locally with 160 characters since it's the default for ansible which I mostly work with.